cluster-autoscaler/cloudprovider/upcloud/hack/version.sh

```
@@ -0,0 +1,22 @@
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+version_file=`dirname "$0"`/../../../version/version.go
+grep '^const ClusterAutoscalerVersion = "' $version_file | cut -d'"' -f2
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)